src/sources/base-source.ts

```diff
@@ -85,5 +85,8 @@ export async function baseSource<UrlParameters extends Record<string, unknown>>(
   if (accessToken) {
     json.accessToken = accessToken;
   }
+  if (schema) {
+    json.schema = schema;
+  }
```

Comment:
/cc @felixpalmer just FYI, this does change the tilejson response slightly, including the schema fields from the map instantiation. I didn't see any other way to expose that, and we appear to need it in fetchMap(). But I'd be open to doing this differently in the future.